src/elements/tabs/tab-group/tab-group.component.ts

```diff
@@ -24,34 +23,11 @@ export type SbbTabChangedEventDetails = {
   previousTab: SbbTabElement | undefined;
 };
 
-export interface InterfaceSbbTabGroupActions {
```

Comment:
To avoid having a breaking change, the interfaces should be kept (but not used) and marked with "@deprecated"